### ISSUE TITLE
Cool new job on mars

### ISSUE THREAD

**Issue by hanayik** [FMRIB-Software-Library/support#14]
we need help developing cutting edge medical imaging scanners on mars.

(this is a demo)

**Comment by nicholst** [comment]
Could we have an automated thing where for jobs or annoucements, after 24 hours or 7 days, say, the issue is closed automatically?

**Comment by hanayik** [comment]
yep. Looks like we can use this github action (https://github.com/marketplace/actions/close-stale-issues). We could set it to only apply to issues labelled with "job". If we did try that, then maybe a longer period would be nice. Perhaps 14 days?




Cheers,

Taylor Hanayik


Taylor Hanayik PhD
Analysis Research Software Engineer
FMRIB, John Radcliffe Hospital
University of Oxford
***@***.***




________________________________

**Comment by nicholst** [comment]
Yeah, 14 days sounds about right... and anyway, it's always searchable.

>
